### ISSUE TITLE
Links in headers 

### ISSUE THREAD

**Issue by agzam** [orgzly/orgzly-android#181]
Would it be possible to render links to appear as "clickable". And not like this:

![screenshot_20170918-105423](https://user-images.githubusercontent.com/1091022/30556545-19370164-9c60-11e7-8171-3ecabb8126b5.jpg)

**Comment by nevenz** [comment]
This is clickable for you in list view, right? You want it to be in the note view too?

I guess it could be the same as for content. Which I plan to turn into "Write" and "Preview" tabs (and additional options like inserting timestamps which you suggested). It would look similar to GitHub comments for example.

**Comment by agzam** [comment]
It makes it difficult when yo get to a note from the notification drawer. Something's deadline triggers notification - and then if it has a header with a link - hard to follow that link

**Comment by nevenz** [comment]
> It makes it difficult when yo get to a note from the notification drawer. Something's deadline triggers notification - and then if it has a header with a link - hard to follow that link

Yeah, makes sense.

**Comment by mjf** [comment]
@nevenz Write/preview tab combo would be splendid! I like the idea and many times though of it (while everytime forgot to create feature request for it). I would like to see write/preview tabs for all text input areas where formatting is enabled.

**Comment by nevenz** [comment]
Toggle button added to action bar instead for now (e93a76ff).
